pkgs/cli/src/main.zig

```diff
@@ -87,12 +92,37 @@ const ZeamArgs = struct {
                 };
             },
         },
+        lean_node: struct {
+            help: bool = false,
+            config_filepath: []const u8 = "./config.yaml",
+            bootnodes_filepath: []const u8 = "./nodes.yaml",
+            validators_filepath: []const u8 = "./validators.yaml",
+            genesis_filepath: ?[]const u8,
+            node_id: u32 = 0,
+            metrics_enable: bool = false,
+            metrics_port: u16 = 9667,
+
+            pub const __shorts__ = .{
+                .help = .h,
+            };
+
+            pub const __messages__ = .{
+                .config_filepath = "Path to the config yaml file",
+                .bootnodes_filepath = "Path to the bootnodes yaml file",
+                .validators_filepath = "Path to the validators yaml file",
+                .genesis_filepath = "Path to the genesis state file",
+                .node_id = "Node id for this lean node",
+                .metrics_port = "Port to use for publishing metrics",
+                .metrics_enable = "Enable metrics endpoint",
+            };
+        },
 
         pub const __messages__ = .{
             .clock = "Run the clock service for slot timing",
             .beam = "Run a full Beam node",
             .prove = "Generate and verify ZK proofs for state transitions on a mock chain",
             .prometheus = "Prometheus configuration management",
+            .lean_node = "Run a lean node",
```

Comment:
rename it to just `node`

and move all other commands to the tools cli as they are all sort of tooling

Reply:
created #177 to make this PR only focus on high priority devnet-0